### ISSUE TITLE
Exported ABC is too fast and short

### ISSUE THREAD

**Issue by NikolaiVChr** [digero/maestro#5]
We have a song, the tempo is set to 87, down from 120.

The duration is then 3.55 in Maestro. But when exporting it gets to be 2.52.

If you need the MIDI to check I can send it to your e-mail.

**Comment by digero** [comment]
Just confirming: the only thing that's wrong is the `%%song-duration` in the .abc file (and in the `T:` of the parts), correct?

As far as I can tell, the song still plays back at the altered tempo you exported; it's just a cosmetic issue that the duration is incorrect.

**Comment by NikolaiVChr** [comment]
No, its not just cosmetic. The song is only 2:52 minutes long when exported.

The midi is 4/4 120bpm 2:04 minutes. So setting the tempo to 87 it gets exported as 2:52 not 3:55.

Unless you mean its a cosmetic issue that it shows 3:55 in Maestro, that might be that Maestro should only show 2:52. Didn't do the math.

**Comment by NikolaiVChr** [comment]
Oh, just did the math. Its the 3:55 that Maestro shows that is wrong. Sorry for the confusion.
